packages/twenty-front/src/modules/command-menu/hooks/useCommandMenuCloseAnimationCompleteCleanup.ts

```diff
@@ -54,6 +58,32 @@ export const useCommandMenuCloseAnimationCompleteCleanup = () => {
 
         emitRightDrawerCloseEvent();
         set(isCommandMenuClosingState, false);
+        set(
+          activeTabIdComponentState.atomFamily({
+            instanceId: WORKFLOW_RUN_STEP_SIDE_PANEL_TAB_LIST_COMPONENT_ID,
+          }),
+          'node',
+        );
+        set(
+          activeTabIdComponentState.atomFamily({
+            instanceId: WORKFLOW_SERVERLESS_FUNCTION_TAB_LIST_COMPONENT_ID,
+          }),
+          'code',
+        );
```

Comment:
I feel this code is unsafe. It's not typed and we use hard coded strings here. I think we should at least use a constant instead of `'node'` and `'code'`. What do you think?